extra_tests/snippets/syntax_assignment.py

```diff
@@ -41,13 +41,15 @@
 assert c == 2
 assert d == 3
 
-a, = [1]
+(a,) = [1]
```

Comment:
Files with `syntax_` prefix is intentionally testing these kind of syntax. Please review those files not to lose this intentions

Reply:
Can't ruff be configured to ignore those lints? That would be preferable in my opinion.

Reply:
Two options here to make immediate progress:

1. We can ignore syntax_*.py in `ruff.toml` entirely
2. We can add ruff directives to each of these `syntax_*.py` files to ignore formatting for the entire file. + add a TODO to refine the directive to the specific line if someone edits the file in the future.

Reply:
1 will be simpler. we have to add every exceptions for new syntax for 2.

Reply:
Cool, this should be resolved here: https://github.com/RustPython/RustPython/pull/5774/commits/9d8c46a5bab7f8d5f41cd6bffe95392929c3cfc0#diff-26f436cb29eecc7bb4f9066747b43df8da81ac808955ce95c109f5d7aa778989